src/main/java/gov/cms/mat/cql_elm_translation/service/HumanReadableService.java

```diff
@@ -142,7 +142,8 @@ HumanReadableMeasureInformationModel buildMeasureInfo(Measure measure) {
                 ? measure.getMeasureMetaData().getSteward().getName()
                 : null)
         .measureTypes(HumanReadableUtil.getMeasureTypes(measure))
-        .stratification(HumanReadableUtil.getStratification(measure))
+        // .stratification(HumanReadableUtil.getStratification(measure))
```

Comment:
We don't need this field ?

Reply:
Looks like HumanReadableMeasureInformationModel.stratification is the cqlDefinitions assigned, and HumanReadableMeasureInformationModel.stratificationDescription holds the information about description.